NetKAN/KHWearableKISProps

```diff
@@ -0,0 +1,20 @@
+{
+    "spec_version" : "v1.0",
+    "ksp_version": "any",
+    "identifier"   : "KHWearableKISProps",
+    "name"         : "Kerbal Hacks: Wearable KIS Props",
+    "$kref"        : "#/ckan/spacedock/1120",
+    "x_netkan_license_ok": true,
```

Comment:
This means that NetKAN inflates the license directly from SpaceDock (iirc) but we don't list license ARR in our [schema](https://github.com/KSP-CKAN/CKAN/blob/master/CKAN.schema#L329) so you should change this line to `"license" : "restricted",` to ensure all your rights are kept in our database.